pkg/Library/Dom/Surfaces/Default/Composer.js

```diff
@@ -26,36 +26,51 @@ export class Composer {
     }
   }
   render(packet) {
-    const {id, container, content: {template, model}} = packet;
-    //log({id, container, model});
-    let slot = this.slots[id];
-    //
+    const {id, container, content: {model}} = packet;
+    if (globalThis.config?.logFlags?.composer) {
```

Comment:
why is the explicit check for `logFlags` needed here?

Reply:
This file has no imports, no dependencies. This is on purpose so we can use it in remote contexts.